core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -1302,6 +1302,24 @@ static PropertyKey create(Template template, Object defaultValue, Object... para
     return key;
   }
 
+  /**
+   * Factory method to create a constant default property
+   * and assign a default value together with its alias.
+   *
+   * @param name String of this property
+   * @param defaultValue Default value of this property in compile time if not null
+   * @param aliases String list of aliases of this property
+   */
+  static PropertyKey create(String name, Object defaultValue, String[] aliases) {
+    PropertyKey key = new PropertyKey(name);
```

Comment:
To make this simple, 
```java
PropertyKey key = create(name, defaultValue);
```
It is a good practice to re-use existing factory methods/constructors as much as possible and leave only the logic that is really different here. It is easier to understand and maintain too.